chat-server/src/middleware/requestOrigin.ts

```diff
@@ -1,6 +1,8 @@
 import { Request, Response, NextFunction } from "express";
 import { getRequestId, logRequest, sendErrorResponse } from "../utils";
 
+export const CUSTOM_REQUEST_ORIGIN_HEADER = "x-request-origin";
```

Comment:
should this be Title-Case, `X-Request-Origin`?

Reply:
Doesn't really matter, headers are case-insensitive